domain-model/src/main/java/uk/gov/hmcts/cmc/domain/utils/ResponseUtils.java

```diff
@@ -84,8 +86,12 @@ public static boolean isFullDefence(Response response) {
         return response.getResponseType().equals(ResponseType.FULL_DEFENCE);
     }
 
-    public static boolean isNoMediation(Response response) {
-        return response.getFreeMediation().filter(Predicate.isEqual(YesNoOption.NO)).isPresent();
+    public static boolean defendantNotOptedForMediation(Response response) {
+        return response.getFreeMediation().filter(Predicate.isEqual(NO)).isPresent();
+    }
+
+    public static boolean defendantOptedForMediation(Response response) {
+        return response.getFreeMediation().filter(Predicate.isEqual(YES)).isPresent();
```

Comment:
can it be 1 method and called something like hasDefendantOptedForMediation?

Reply:
actioned